sound/soc/sof/intel/hda-dai.c

```diff
@@ -344,16 +344,6 @@ static int hda_link_pcm_trigger(struct snd_pcm_substream *substream,
 	w = snd_soc_dai_get_widget(dai, substream->stream);
 
 	switch (cmd) {
-	case SNDRV_PCM_TRIGGER_RESUME:
-		/* set up hw_params */
-		ret = hda_link_pcm_prepare(substream, dai);
-		if (ret < 0) {
-			dev_err(dai->dev,
-				"error: setting up hw_params during resume\n");
-			return ret;
-		}
-
-		fallthrough;
```

Comment:
didn't we merge this already? should rebase, no?

Reply:
yes, rebased now